cypress/support/accessibility/a11y-utils.ts

```diff
@@ -130,11 +133,12 @@ export default (from, end) => {
               force: true,
             });
           }
+
           cy.injectAxe()
+            // @ts-ignore
             .wait(250)
-            .then(() => {
-              cy.checkA11y(null, A11YOptions, terminalLog);
-            });
+            // @ts-ignore
```

Comment:
Why did this need `@ts-ignore` twice here?

Reply:
not sure, but anyways we will delete this file after the refactor :)